src/fsharp/FSharp.Compiler.Private/FSharp.Compiler.Private.fsproj

```diff
@@ -16,26 +19,6 @@
     <Tailcalls>true</Tailcalls> <!-- .tail annotations always emitted for this binary, even in debug mode -->
   </PropertyGroup>
 
-  <Target Name="CopyToBuiltBin" BeforeTargets="BuiltProjectOutputGroup" AfterTargets="CoreCompile">
```

Comment:
This target needs to stay in this file because this is where the package references live.  It's simply an implementation detail that they also end up next to fsc in the output and if that ever changes we'll be in for a world of hurt trying to figure out what changed.

I'm guessing this changed because building only `netstandard2.0` here means not all of the files are present, e.g., `Microsoft.Build.dll`.  If that's the case, those `BuildProjectOutputGroupKeyOutput` items could be made to behave better with `Condition="'$(TargetFramework)' == 'net472'"`

Reply:
ok i see thanks (out of curiosity could you also put package references in FSC?)